client/src/components/AddNewProfile.js

```diff
@@ -16,46 +17,96 @@ const AddNewProfile = () => {
     support_type: "",
     profile_type: "",
     status: "new",
-    join_date: new Date()
+    nationality: "",
+    join_date: new Date(),
   });
-
+  const [errors, setErrors] = useState([]);
   const [profileCreated, setProfileCreated] = useState(null);
+  const [nationality, setNationality] = useState([]);
 
-  const handleChange = event => {
+  const handleChange = (event) => {
     updateField(event.target.name, event.target.value);
   };
 
   const createProfile = () => {
-    postProfile(profileData).then(isSuccessful => {
+    let errs = [];
+    if (profileData.firstname.length === 0) {
+      errs.push(
+        <li>
+          <p>First name cannot be empty</p>
+        </li>
+      );
```

Comment:
```suggestion
      errs.push('First name cannot be empty');
```
Since we are repeating the `<li><p>` 4 times here, let's remove it from here and from the other `errs.push` and put it in the `return` section